### ISSUE TITLE
Web service responses cached by Safari

### ISSUE THREAD

**Issue by gcbartlett** [dockersamples/k8s-wordsmith-demo#8]
Under Safari Technology Preview Release 61 (Safari 12.0, WebKit 13606.1.25.1), **noun** and **adjective** APIs are only called once each, and so the same words are displayed.  I suspect this is because the responses from the **web** server do not contain any cache control headers, and so Safari reuses the first response.

<img width="832" alt="image" src="https://user-images.githubusercontent.com/4016279/43358384-7897f038-9245-11e8-8a53-36b5cf95d768.png">

**Comment by gcbartlett** [comment]
I tried adding cache control headers to `dispatcher.go`:
```
	w.Header().Set("source", ip)
	w.Header().Set("Cache-Control", "no-cache, no-store, must-revalidate")
	w.Header().Set("Pragma", "no-cache")
	w.Header().Set("Expires", "0")
```
but that didn't help.

But, adding a different parameter to the requested URLs in `app.js` did the job:
```
  getWord($http, $timeout, '/words/noun&word=1', function(resp) {
...
  getWord($http, $timeout, '/words/noun&word=2', function(resp) {
...
  getWord($http, $timeout, '/words/adjective&word=1', function(resp) {
...
  getWord($http, $timeout, '/words/adjective&word=2', function(resp) {
```

**Comment by ManoMarks** [comment]
@gcbartlett sorry just checking out this issue. If you want to submit a PR that would help us a lot.

**Comment by otherguy** [comment]
I fixed this issue (and more) in #16. Would appreciate it if you could merge, @ManoMarks

**Comment by otherguy** [comment]
@BretFisher I replaced my old PR with #19. This fixed the issue described here.
